src/PositionManager.sol

```diff
@@ -138,9 +140,9 @@ contract PositionManager is IPositionManager, ERC721Permit, PoolInitializer, Mul
         }
         _mint(owner, tokenId);
 
-        (BalanceDelta delta,) = _modifyLiquidity(range, liquidity.toInt256(), bytes32(tokenId), hookData);
+        (BalanceDelta delta,) = _modifyLiquidity(poolPos, liquidity.toInt256(), bytes32(tokenId), hookData);
 
-        tokenRange[tokenId] = range;
+        poolPositions[tokenId] = poolPos.toId();
```

Comment:
definitely need an event logging all of the fields here imo? Because all that people can look up is a hash